sdk/src/types/block/context_input/mod.rs

```diff
@@ -135,7 +135,7 @@ mod tests {
     fn reward() {
         let reward: ContextInput = serde_json::from_value(serde_json::json!(
             {
-                "type": 3,
+                "type": 2,
```

Comment:
I noticed t hat the supposed-to-be-failing test down below uses `serde_json::from_str(<raw_string>)` instead of how we do it above? This could be made consistent,right?

Reply:
Done